ert3/evaluator/_evaluator.py

```diff
@@ -1,57 +1,136 @@
 import pickle
-from typing import Dict
+import time
+from typing import Dict, Optional, Union, Any
+from concurrent import futures
+import sys
 
 import ert
 from ert_shared.ensemble_evaluator.config import EvaluatorServerConfig
+from ert_shared.models.base_run_model import BaseRunModel
+from ert_shared.status.tracker.evaluator import EvaluatorTracker
+from ert_shared.cli.monitor import Monitor
 from ert_shared.ensemble_evaluator.ensemble.base import Ensemble
-from ert_shared.ensemble_evaluator.entity.identifiers import EVTYPE_EE_TERMINATED
+from ert_shared.ensemble_evaluator.entity.identifiers import (
+    EVTYPE_EE_TERMINATED,
+)
 from ert_shared.ensemble_evaluator.evaluator import EnsembleEvaluator
 
 from ert_shared.status.entity.state import (
     ENSEMBLE_STATE_STOPPED,
     ENSEMBLE_STATE_FAILED,
-    REALIZATION_STATE_FINISHED,
 )
 
 
+class DummyBaseRunModel(BaseRunModel):
```

Comment:
Maybe rename to `DummyModel` or `ERT3Model` and remove the inheritance? It will give you tons of stuff not used.

And we're not going to use `BaseRunModel` for updates in ERT3 anyway (I hope 🤞 )

Reply:
> Maybe rename to `DummyModel` or `ERT3Model` and remove the inheritance? It will give you tons of stuff not used.
> 
> And we're not going to use `BaseRunModel` for updates in ERT3 anyway (I hope 🤞 )

The same as above: 
yes, as long as we still use `BaseRunModel`, some functions need to be implemented; eg. `runSimulations` and `teardown_context`.  Also unfortunately it requires the inheritance. But @Blunde1 is working on BaseRunModel removal / substitution by some basic `jobstate` class.